SB-1.md

```diff
@@ -0,0 +1,76 @@
+# SB-1 (AMQP Backed Implementation)
+
+**Status**: Under Review
+
+This proposal introduces a method of using a message queue (Anything implementing
+AMQP 0.9.1, focusing on RabbitMQ) rather than the internal, custom queue.
+
+## Context
+
+Much of Seabird core has been built around a custom message queue, limited to a
+single node. This has made more complex operations harder to implement and led
+to less robust code.
+
+## Goals
+
+- Remove custom internal message queue
```

Comment:
I think it'd be good to call out explicitly what this means. Given that you're still planning to have wrappers around parts of the subscription it seems like parts will still be there (not really of the message queue, but of the custom code which seems the spirit of this RFC).